src/test/testGraphs.ts

```diff
@@ -1,18 +1,32 @@
-import { Graph, HexString } from "../utilities/types";
+import { DSNPUserId } from "@dsnp/sdk/dist/types/core/identifiers";
+import { HexString } from "../utilities/types";
 import { generateDsnpUserId, getPrefabDsnpUserId } from "./testAddresses";
 
+type Graph = Record<DSNPUserId, Record<DSNPUserId, boolean>>;
```

Comment:
can you put some more explanatory comments in here? I think a Graph indicates action direction, e.g. in the case of "follower" it means the first DSNPUserId is Following DSNPUserId 2, === true, but I'm not sure.

Reply:
sure.